src/OrchardCore.Themes/TheAgencyTheme/Assets/package.json

```diff
@@ -40,7 +40,8 @@
     },
     "devDependencies": {
         "autoprefixer": "10.4.4",
-        "browser-sync": "2.27.9",
+        "browser-sync": "2.29.3",
```

Comment:
Is there any particular reason browser-synch is not updated in the two other themes?